app/code/Magento/Inventory/Model/ValidationChecker/NoSpecialCharsInString.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model\ValidationChecker;
```

Comment:
Validation checker... Sounds like "weapon shooter". Is it possible to give it more appropriate namespace?